test/integration-test-v2.py

```diff
@@ -280,6 +284,33 @@ def test_loadgeneration():
         shell=True,
         stderr=subprocess.STDOUT)
 
+def test_sct_embedding():
+    order = auth_and_issue([random_domain()])
+    cert = x509.load_pem_x509_certificate(str(order.fullchain_pem), default_backend())
+
+    # make sure there is no poison extension
+    try:
+        cert.extensions.get_extension_for_oid(x509.ObjectIdentifier("1.3.6.1.4.1.11129.2.4.3"))
+        raise Exception("certificate contains CT poison extension")
+    except x509.ExtensionNotFound:
+        # do nothing
+        print("no poison extension")
```

Comment:
Nit: Could you make this a `pass` instead of a `print`? I don't think it will be useful output.